### PR TITLE
Accounts-V2: Add tests for exporting and importing

### PR DESCRIPTION
**Pull request by 0xKiwi**
**What type of PR is this?**
Test additions

**What does this PR do? Why is it needed?**
This PR adds testing to the import and export functions for the account util of the validator client.

### CODE REVIEW

**Comment by rauljordan** [comment]
@0xKiwi I highly recommend testing the actual import command by creating a new cliCtx group: https://github.com/prysmaticlabs/prysm/pull/6611/files#diff-edd59472887f8028ffdf8f734461cc40R32 and also using Victor's new test assertion helpers, it will make the tests easier

File: validator/accounts/v2/export.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/prysmaticlabs/prysm/shared/params"
+
```
Comment (rauljordan) [inline validator/accounts/v2/export.go:14]:
```suggestion
```

File: validator/accounts/v2/export.go [hunk 2]
```diff
@@ -25,7 +27,7 @@ func ExportAccount(cliCtx *cli.Context) error {
 	// Read a wallet's directory from user input.
 	walletDir, err := inputWalletDir(cliCtx)
 	if errors.Is(err, ErrNoWalletFound) {
-		return errors.New("no wallet found, create a new one with ./prysm.sh validator wallet-v2 create")
+		log.Fatal("no wallet found, create a new one with ./prysm.sh validator wallet-v2 create")
```
Comment (rauljordan) [inline validator/accounts/v2/export.go:29]: Why not return error as we discussed?

File: validator/accounts/v2/export.go [hunk 3]
```diff
@@ -59,6 +64,8 @@ func ExportAccount(cliCtx *cli.Context) error {
 	if err := logAccountsExported(wallet, accounts); err != nil {
 		log.WithError(err).Fatal("Could not log out exported accounts")
 	}
+
+	fmt.Println("fuck6")
```
Comment (rauljordan) [inline validator/accounts/v2/export.go]:
```suggestion
```

File: validator/accounts/v2/export_test.go [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+	"flag"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/shared/testutil/assert"
+	"github.com/prysmaticlabs/prysm/validator/flags"
+	"github.com/urfave/cli/v2"
+
```
Comment (rauljordan) [inline validator/accounts/v2/export_test.go:14]:
```suggestion
```

File: validator/accounts/v2/export_test.go [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+	ctx := context.Background()
+	err := initializeDirectWallet(walletDir, passwordsDir)
+	assert.NoError(t, err)
+	cfg := &WalletConfig{
+		WalletDir:      walletDir,
+		PasswordsDir:   passwordsDir,
+		KeymanagerKind: v2.Direct,
+	}
+	w, err := NewWallet(ctx, cfg)
+	if err != nil {
```
Comment (rauljordan) [inline validator/accounts/v2/export_test.go:31]: No need for this, you can do `require.NoError(t, err)` which does the same as if err != nil { t.Fatal(err

File: validator/accounts/v2/export_test.go [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+		t.Fatal(err)
+	}
+
+	assert.NoError(t, err)
```
Comment (rauljordan) [inline validator/accounts/v2/export_test.go:35]: Remove, use require.NoError instead

File: validator/accounts/v2/export_test.go [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+	keymanager, err := w.InitializeKeymanager(ctx, true)
+	if err != nil {
```
Comment (rauljordan) [inline validator/accounts/v2/export_test.go:37]: Remove

File: validator/accounts/v2/export_test.go [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+		assert.NoError(t, os.RemoveAll(passwordsDir))
+		assert.NoError(t, os.RemoveAll(exportDir))
+		assert.NoError(t, os.RemoveAll(importDir))
+	}()
+	wallet := setupWallet(t, testDir)
+
+	accounts, err := wallet.AccountNames()
+	assert.NoError(t, err)
+
+	if len(accounts) == 0 {
```
Comment (rauljordan) [inline validator/accounts/v2/export_test.go:67]: You can do require.Equal(t, len(accounts), 0)

File: validator/accounts/v2/wallet.go [hunk 9]
```diff
@@ -11,6 +11,8 @@ import (
 	"path"
 	"strings"
 
+	"github.com/prysmaticlabs/prysm/shared/params"
+
```
Comment (rauljordan) [inline validator/accounts/v2/wallet.go:15]:
```suggestion
```

File: validator/accounts/v2/export.go [hunk 10]
```diff
@@ -45,12 +46,15 @@ func ExportAccount(cliCtx *cli.Context) error {
 
 	allAccounts, err := wallet.AccountNames()
 	if err != nil {
-		log.WithError(err).Fatal("Could not get account names")
+		log.WithError(err).Error("Could not get account names")
```
Comment (terencechain) [inline validator/accounts/v2/export.go:49]: Why not return error?

And it feels out of scope of current PR which titled "Add tests... "  : )

Comment (0xKiwi) [reply validator/accounts/v2/export.go:49]: Oh whoops that was meant for debugging, had some weird stuff with the fatals. Thanks for pointing it out!

File: validator/accounts/v2/export.go (hunk 2 above)
Comment (0xKiwi) [reply validator/accounts/v2/export.go:29]: Was waiting for the wrapper you mentioned to be implemented, maybe it can be done for all of accounts-v2 in a PR?

File: validator/accounts/v2/export_test.go (hunk 8 above)
Comment (farazdagi) [reply validator/accounts/v2/export_test.go:59]: The order of arguments is `expected, got` in equal. So, something like:
```golang
require.Equal(t, 0, len(accounts), "Expected more accounts") // the last param is optional
```
Please see my comment on https://github.com/prysmaticlabs/prysm/pull/6626 for my rationale.

Comment (farazdagi) [reply validator/accounts/v2/export_test.go:59]: Dismissing my suggestion (as it is too fragile to require the order, and not that necessary): https://github.com/prysmaticlabs/prysm/pull/6631
